src/app/features/increase-fee-drawer/increase-fee-drawer.tsx

```diff
@@ -53,7 +68,7 @@ export function IncreaseFeeDrawer() {
               If your transaction has been pending for a long time, its fee might not be high enough
               to be included in a block. Increase the fee and try again.
             </Caption>
-            <IncreaseFeeForm />
+            {btcTx ? <IncreaseBtcFeeForm btcTx={btcTx} /> : <IncreaseFeeForm />}
```

Comment:
Is this component handling both btc and stx? If we add another chain, would we have to continually extend this ternary?

Is there a nicer way to compose so we have something more like

```ts
<IncreaseBtcFeeDrawer>
  // interally using the generic component 
  <GenericFeeDrawer />
</IncreaseBtcFeeDrawer>

<IncreaseStxFeeDrawer>
  <GenericFeeDrawer />
</IncreaseStxFeeDrawer>
```

Reply:
done